docs/commands/context.md

```diff
@@ -63,6 +63,30 @@ tracking the evolution of arbitrary locations in memory. Tracked locations can
 be removed one by one using `memory unwatch`, or altogether with `memory reset`.
 
 
+To have the stack grow up enable the `context.grow_stack_up`, i.e., largest stack values are on top.
```

Comment:
Thank you for taking the time to make all the suggestions. I took care of the first two.  

As for the grow down/up, that's how I was taught and how I see it quite a bit.  Yes, it means that push is to the bottom.  Just think, as weird as it is to you, it's actually the same to me in the opposite direction.  Because both methods have their pros and cons, I made it a preference setting and kept it defaulting to the way it was.

I originally had the language grow down, but changed it to reflect how the addresses appear on the screen instead of the stacks growth.  Should I change the references to it to reflect the stack growing down?

Reply:
I am fine with having a different way to represent the stack (even though I think it looks dumb upside down :p ). I know many people prefer it your way. I just want the _wording_ to be consistent, and 'grows down' is how I've always seen.

No matter which way the stack is drawn, pushing values on it makes it taller. Let's let @hugsy weigh in.